tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/story_participant.tsx

```diff
@@ -0,0 +1,8 @@
+import { CheckboxInput, FeatureToggle } from '../base';
+
+export const story_pref: FeatureToggle = {
+  name: 'Be story participant',
```

Comment:
This feels like a pretty huge thing to lock behind a small preference. I think it should be visible in the antagonist menu with everything else even if it does not contribute to Dynamic.